.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Please, before opening PR !always! pull changes from dev branch to resolve merge conflicts and have last changes.
Currently in dev branch .gitigrone has .idea/ and like this your .idea would not appear in the PR.
So, please, pull changes from dev, push them and remove .idea files from PR